__tests__/app.test.js

```diff
@@ -131,4 +131,72 @@ describe('GET api/articles', () => {
+});
+
+describe('GET api/articles/:article_id/comments', () => {
+    test('sending an API request to api/articles/:articles_id/comments returns a Status-200: responds with an object, with a key comments, which has the value of an array of comments relevant to given id - in this case, article id 1 should have 11 comments.', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                const comments = body.comments
+                expect(comments).toHaveLength(11)
+            })
+    })
+    test('sending an API request to api/articles/:articles_id/comments returns a Status-200: responds with an object, with a key comments, which has the value of an array of comments relevant to given id - in this case, article id 3 should have 2 comments. Each comment has the required properties', () => {
+        return request(app)
+            .get('/api/articles/3/comments')
+            .expect(200)
+            .then(({body}) => {
+                const comments = body.comments
+                expect(comments).toHaveLength(2)
+                comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        "comment_id" : expect.any(Number),
+                        "votes" : expect.any(Number),
+                        "created_at" : expect.any(String),
+                        "author" : expect.any(String),
+                        "body" : expect.any(String),
+                        "article_id" : 3
+                    })
+                })
+            })
+    })
+    test('sending an API request to api/articles/:articles_id/comments returns a Status-200: responds with an object, with a key comments, which has the value of an array of comments relevant to given id - in this case, article id 1 should have 11 comments. Each comment has the required properties. The returned array is returned with its most recent comment 1st', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                const comments = body.comments
+                expect(comments).toHaveLength(11)
+                comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        "comment_id" : expect.any(Number),
+                        "votes" : expect.any(Number),
+                        "created_at" : expect.any(String),
+                        "author" : expect.any(String),
+                        "body" : expect.any(String),
+                        "article_id" : 1
+                    })
+                })
+                expect(comments[0].comment_id).toBe(5)
+                expect(comments[10].comment_id).toBe(9)
+            })
+    })
+    test.only('sending an API request to an article_id that returns no results returns a Status-404, and an object with msg key "bad request"', () => {
```

Comment:
404 is not found, 400 is a bad request - in this case, think about when you read an article on bbc news - if the article has no comments, you dont want to see an error just because nobody has commented on it right? An error will prevent us from loading the page and reading the article - in which case no new article would ever receive a comment because we'd never be able to read it in the first place! So you want to say the request for the comments was successful - you just didnt get any comments back